### PR TITLE
Translation

### PR DESCRIPTION
**Pull request by shanuka**
Do it the standard way provided by Flutter. This needs to be set up.
https://docs.flutter.dev/ui/accessibility-and-internationalization/internationalization#adding-your-own-localized-messages

Use the user's device language to pick from EN, FR or NL automatically
If their language is not one of the three configured languages, fall back to EN
Provide a way to switch languages from the settings page
Cupertino Picker for the language picker: https://api.flutter.dev/flutter/cupertino/CupertinoPicker-class.html

### CODE REVIEW

File: pubspec.yaml [hunk 1]
```diff
@@ -140,7 +143,9 @@ flutter:
     - packages/smartcontracts/contracts/apps/SimpleFaucet.abi.json
     - packages/smartcontracts/contracts/standards/ERC20.abi.json
     - packages/smartcontracts/contracts/apps/Profile.abi.json
-  uses-material-design: false
+   
+  uses-material-design: true
```
Comment (kev-techi) [inline pubspec.yaml:147]: why do we need to enable material design? this will add additional libraries for no reason.

Comment (kev-techi) [reply pubspec.yaml:147]: this only affects including material icons in the app. We don't use material icons.

File: lib/main.dart [hunk 2]
```diff
@@ -148,6 +180,18 @@ class MyAppState extends State<MyApp> {
             routerConfig: router,
             theme: theme,
             title: '$titlePrefix Wallet',
+            locale: _locale,
+            localizationsDelegates: [
+              AppLocalizations.delegate, // Add this line
+              GlobalMaterialLocalizations.delegate,
```
Comment (kev-techi) [inline lib/main.dart:186]: similar comment with regards to material. We don't use or need material

**Review by kev-techi — CHANGES_REQUESTED** [review]
Looks mostly good. Just some comments about material

File: pubspec.yaml (hunk 1 above)
Comment (shanuka) [reply pubspec.yaml]: removed

File: lib/main.dart (hunk 2 above)
Comment (shanuka) [reply lib/main.dart]: removed

**Review by kev-techi — APPROVED** [review]

**Pull request merged**